simple_kafka_client/SimpleKafkaClient.php

```diff
@@ -0,0 +1,51 @@
+<?php
+declare(strict_types=1);
+
+use SimpleKafkaClient\Exception;
+use SimpleKafkaClient\Metadata;
+use SimpleKafkaClient\Topic;
+use SimpleKafkaClient\TopicPartition;
+
+abstract class SimpleKafkaClient
+{
+    /**
+     * @param bool $allTopics
+     * @param int $timeoutMs
+     * @param Topic $topic
+     * @return Metadata
+     * @throws Exception
+     */
+    public function getMetadata(bool $allTopics, int $timeoutMs, Topic $topic): Metadata {}
+
+    /**
+     * @return int
+     */
+    public function getOutQLen(): int {}
+
+    /**
+     * @param int $timeoutMs
+     * @return int
+     */
+    public function poll(int $timeoutMs): int {}
+
+    /**
+     * $low and $high will be passed as reference and will contain
```

Comment:
I am afraid this will be skipped by developpers

The best would be to add the text after the $low and $high